ui/src/cluster-workflow-templates/cluster-workflow-template-markdown.scss

```
@@ -1,8 +1,15 @@
 @import 'node_modules/argo-ui/src/styles/config';
 
 .wf-rows-name {
-  line-height: 1.5em;
-  display: inline-block;
-  vertical-align: middle;
+  line-height: 2.5em;
   white-space: pre-line;
+  text-wrap: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  padding: 0.25em 0;
+}
+
+.wf-rows-name > * {
+  overflow: hidden;
+  text-overflow: ellipsis;
```

You can simplify here and or in others `wf-rows-name` usage with &

```suggestion
  & > * {
    overflow: hidden;
    text-overflow: ellipsis;
```